docs/operation/account_administration.rst

```diff
@@ -0,0 +1,86 @@
+
+Account Administration
+======================
+
+This section highlights **MetalK8s Account Administration** which covers
+changing the default username and password for some MetalK8s services.
+
+
+Administering Grafana
+*********************
+
+A fresh install of MetalK8s has a Grafana service instance with default
+credentials: ``admin`` / ``admin``. For more information on how to access
+Grafana, please refer to :ref:`this procedure <quickstart-services-grafana>`
+
+Changing Grafana username and password
+--------------------------------------
+
+To change the default username and password for Grafana on a MetalK8s cluster,
+perform the following procedures:
+
+#. Generate a new Grafana username and password using a **Base64 encoder**
+   and note it down.
+
+   .. code-block:: shell
+
+      $ echo -n 'new-username' | base64
+      $ echo -n 'new-password' | base64
+
+#. Create a file named ``patch-secret.yaml`` that has following content:
+
+   .. code-block:: yaml
+
+      data:
+        admin-password: bmV3cGFzc3dvcmQ=   <-- Replace with base64 password
+        admin-user: bmV3cGFzc3dvcmQ=       <-- Replace with base64 username
+        ldap-toml: ""
+
+#. Apply the patch file by running:
+
+   .. code-block:: shell
+
+      $ kubectl patch secrets prometheus-operator-grafana --patch "$(cat patch-secret.yml)" -n metalk8s-monitoring
+
+#. Verify that the secret was updated successfully.
+
+   .. code-block:: shell
+
+      $ kubectl get secrets prometheus-operator-grafana --output yaml -n metalk8s-monitoring
+
+#. Now, roll out the new updates for the grafana deployment.
+
+   .. code-block:: shell
+
+      $ kubectl rollout restart deploy prometheus-operator-grafana -n metalk8s-monitoring
+
+#. Access the Grafana instance and authenticate yourself using the new Account
+   credentials.
+
+
+
+Administering MetalK8s GUI
```

Comment:
This section needs rewording: we are actually changing the Kubernetes API user/password here. Which implies not only changes to the GUI, but also to SaltAPI credentials (we could explain these implications in this document, sure).

Reply:
Looks like SaltAPI authentication was moved to certificate based auth in the `2.4` branch so, i doubt this has an impact.

Reply:
https://github.com/scality/metalk8s/pull/1944#issuecomment-544963797 ??

Reply:
I don't think the UI still work when you update the login/password while playing with it. There will be probably some work to do on the UI to force user to log in again by going to the '/login'